### PR TITLE
ci: make Visual Regression Testing work again

### PR DESCRIPTION
**Pull request by 2020Deeya**
# Description

In order to make the Visual regression testing work in pipeline:

- Adjust the threshold for now

## Links

https://jira-eng-gpk2.cisco.com/jira/browse/MOMENTUM-271

### CODE REVIEW

File: packages/components/src/components/avatar/avatar.e2e-test.ts [hunk 1]
```diff
@@ -28,11 +28,11 @@ test.fixme('mdc-avatar', async ({ componentsPage }) => {
    */
 
   // TODO: fix visual regression test on CI
```
Comment (cscheffauer) [inline packages/components/src/components/avatar/avatar.e2e-test.ts:30]: suggestion: remove TODO comment as well given it works for now on CI (applies for all e2e tests)

File: packages/components/config/playwright/setup/constants.ts [hunk 2]
```diff
@@ -15,8 +15,8 @@ const ACCESSIBILITY = {
 };
 
 const VISUAL_REGRESSION = {
-  THRESHOLD: 0,
-  MAX_DIFF_PIXELS_RATIO: 0.03,
+  THRESHOLD: 0.1,
```
Comment (cscheffauer) [inline packages/components/config/playwright/setup/constants.ts:18]: maybe worth adding a comment here that we keep the threshold, max_diff_pixels_ratio at a higher value until we have pixel perfect visual regression testing enabled.

**Review by cscheffauer — CHANGES_REQUESTED** [review]

File: packages/components/config/playwright/setup/constants.ts (hunk 2 above)
Comment (2020Deeya) [reply packages/components/config/playwright/setup/constants.ts:19]: Sure, it'll keep other developers updated.

File: packages/components/src/components/avatar/avatar.e2e-test.ts (hunk 1 above)
Comment (2020Deeya) [reply packages/components/src/components/avatar/avatar.e2e-test.ts]: Done.

**Review by cscheffauer — APPROVED** [review]

**Pull request merged**